docs/roles.md

```diff
@@ -5,7 +5,22 @@ The contracts in this repository define a number of different roles that have th
 
 ## `HATVaultsRegistry.owner`
 
-- `swapAndSend` and swap and send the vauit's token that are earmarked for payout (after approveClaim)
+- can call `setFeeSetter` set the feeSetter role
```

Comment:
Can you undo these changes?

Reply:
this is a merge going wrong, we can update later

Reply:
Ok yeah something went wrong with the conflicts, I just made it as it is now